gtda/diagrams/preprocessing.py

```diff
@@ -368,6 +371,9 @@ def fit(self, X, y=None):
             Input data. Array of persistence diagrams, each a collection of
             triples [b, d, q] representing persistent topological features
             through their birth (b), death (d) and homology dimension (q).
+            It is important that, for each possible homology dimension, the
```

Comment:
This should be added to all transformers taking a diagram as an input.

Reply:
Indeed, and this is why #233 remains open. I just didn't want to mix issues too much, but wanted to make a test run here.